### PR TITLE
Immudb version update to 1.3.0

### PR DESCRIPTION
**Pull request by pranavv0**
Update:
1. Image updated from 1.0.5 to 1.3.0
2. immudb-config-generator updated with changed in SQL syntax
3. Update in immudb-config-generator yaml file

### CODE REVIEW

File: Docker-Swarm-deployment/single-node/immudb/docker/immudb-config-generator/immudb-config-generator.py [hunk 1]
```diff
@@ -21,37 +23,44 @@
...
+
+client.useDatabase("iudxauth")
+client.sqlExec("CREATE TABLE table_auditing(id VARCHAR[250] NOT NULL, body VARCHAR[250] NOT NULL,userid VARCHAR[250] NOT NULL,endpoint VARCHAR[250] NOT NULL,method VARCHAR[250] NOT NULL,time INTEGER NOT NULL,PRIMARY KEY id);")
+
+client.useDatabase("iudxrsorg")
+client.sqlExec("CREATE TABLE IF NOT EXISTS rsauditingtable (id VARCHAR[128] NOT NULL, api VARCHAR[128], userid VARCHAR[128],epochtime INTEGER,resourceid VARCHAR[256], isotime VARCHAR[64], providerid VARCHAR[128], PRIMARY KEY id);")
+
+client.sqlExec("CREATE INDEX ON rsauditingtable(userid, epochtime, providerid, resourceid);")
+
+client.sqlExec("CREATE TABLE rsaudit (id VARCHAR[128] NOT NULL,api VARCHAR[128] NOT NULL,userid VARCHAR[128] NOT NULL,epochtime INTEGER NOT NULL,resourceid VARCHAR[256] NOT NULL,isotime VARCHAR[64] NOT NULL,providerid VARCHAR[128] NOT NULL,size INTEGER, PRIMARY KEY id);")
```
Comment (swaminathanvasanth) [inline Docker-Swarm-deployment/single-node/immudb/docker/immudb-config-generator/immudb-config-generator.py:51]: Change size to responsesize

**Comment by abhi4578** [comment]
@gopal-mahajan , is the attribute name 'size' in [rsaudit table](https://github.com/datakaveri/iudx-resource-server/pull/272/commits/5a0ebd9981d5a4912ad8e84d6b7ec8143f06d988)  or  'responsize'

**Comment by gopal-mahajan** [comment]
> @gopal-mahajan , is the attribute name 'size' in [rsaudit table](https://github.com/datakaveri/iudx-resource-server/pull/272/commits/5a0ebd9981d5a4912ad8e84d6b7ec8143f06d988) or 'responsize' ?

column name should be size as discussed with @swaminathanvasanth

File: Docker-Swarm-deployment/single-node/immudb/docker/immudb-config-generator/immudb-config-generator.py (hunk 1 above)
Comment (swaminathanvasanth) [reply Docker-Swarm-deployment/single-node/immudb/docker/immudb-config-generator/immudb-config-generator.py:49]: The API servers are changed to insert as size. Lets not change the schema. We will keep it as size

Comment (abhi4578) [reply Docker-Swarm-deployment/single-node/immudb/docker/immudb-config-generator/immudb-config-generator.py:49]: okay.

**Comment by swaminathanvasanth** [comment]
The implementation was changed before I could discuss with @gopal-mahajan 

Now all the API servers will use "size" as the column name for response size. 
Lets keep the table column name as "size" itself. @karun-singh @abhi4578

**Comment by abhi4578** [comment]
Is the column with varchar data types in various table have sufficient characters allocated? especially table used by auth with attribute body[2048]? Can in some cases the characters be reduced or should be increased ? @gopal-mahajan 
```
client.sqlExec("CREATE TABLE auditingtable(id VARCHAR[128] NOT NULL, userRole VARCHAR[64] NOT NULL,userID VARCHAR[128] NOT NULL,iid VARCHAR[250] NOT NULL,api VARCHAR[128] NOT NULL,method VARCHAR[32] NOT NULL,time INTEGER NOT NULL,iudxID VARCHAR[256] NOT NULL,PRIMARY KEY id);")
client.sqlExec("CREATE INDEX ON auditingtable(userID, iudxID, time, api);")
client.useDatabase("iudxauth")
client.sqlExec("CREATE TABLE table_auditing(id VARCHAR[128] NOT NULL, body VARCHAR[2048] NOT NULL,userid VARCHAR[128] NOT NULL,endpoint VARCHAR[128] NOT NULL,method VARCHAR[32] NOT NULL,time INTEGER NOT NULL,PRIMARY KEY id);")

client.sqlExec("CREATE INDEX ON table_auditing(userid, endpoint, time);")
client.useDatabase("iudxrsorg")


client.sqlExec("CREATE TABLE rsaudit (id VARCHAR[128] NOT NULL,api VARCHAR[128] NOT NULL,userid VARCHAR[128] NOT NULL,epochtime INTEGER NOT NULL,resourceid VARCHAR[256] NOT NULL,isotime VARCHAR[64] NOT NULL,providerid VARCHAR[128] NOT NULL,size INTEGER, PRIMARY KEY id);")
client.sqlExec("CREATE INDEX ON rsaudit(userid, epochtime, providerid, resourceid);")

```

**Comment by abhi4578** [comment]
> Is the column with varchar data types in various table have sufficient characters allocated? especially table used by auth with column body[2048]? Can in some cases the characters be reduced or should be increased ? @gopal-mahajan . The 1.3.0 version forces to make varchar len bounded, and therefore need to carefully allocate the sizes for the columns. It should be sufficient enough so that insert with big column values would not cause error) or that total index length is less than 1k bytes, index length=sum of lengths of each index field allocated bytes.
> 
> ```
> client.sqlExec("CREATE TABLE auditingtable(id VARCHAR[128] NOT NULL, userRole VARCHAR[64] NOT NULL,userID VARCHAR[128] NOT NULL,iid VARCHAR[250] NOT NULL,api VARCHAR[128] NOT NULL,method VARCHAR[32] NOT NULL,time INTEGER NOT NULL,iudxID VARCHAR[256] NOT NULL,PRIMARY KEY id);")
> client.sqlExec("CREATE INDEX ON auditingtable(userID, iudxID, time, api);")
> client.useDatabase("iudxauth")
> client.sqlExec("CREATE TABLE table_auditing(id VARCHAR[128] NOT NULL, body VARCHAR[2048] NOT NULL,userid VARCHAR[128] NOT NULL,endpoint VARCHAR[128] NOT NULL,method VARCHAR[32] NOT NULL,time INTEGER NOT NULL,PRIMARY KEY id);")
> 
> client.sqlExec("CREATE INDEX ON table_auditing(userid, endpoint, time);")
> client.useDatabase("iudxrsorg")
> 
> 
> client.sqlExec("CREATE TABLE rsaudit (id VARCHAR[128] NOT NULL,api VARCHAR[128] NOT NULL,userid VARCHAR[128] NOT NULL,epochtime INTEGER NOT NULL,resourceid VARCHAR[256] NOT NULL,isotime VARCHAR[64] NOT NULL,providerid VARCHAR[128] NOT NULL,size INTEGER, PRIMARY KEY id);")
> client.sqlExec("CREATE INDEX ON rsaudit(userid, epochtime, providerid, resourceid);")
> ```
unbound varchar possible in immudb if not indexed

 - body in table_auditing is now unbounded.
 - Also tested with by puttiung 4k chars in the insert, it works

**Review by abhi4578 — APPROVED** [review]

**Pull request merged**